modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Filters.tsx

```diff
@@ -499,12 +557,11 @@ function AddFDSFilterModalContent({
 										</label>
 
 										<ClayRadioGroup
-											defaultValue={
-												formValues[
-													includeModeFormElementId
-												]
-											}
 											name={includeModeFormElementId}
+											onChange={(newVal: any) => {
+												setInclude(newVal === 'true');
+											}}
+											value={include ? 'true' : 'false'}
```

Comment:
That may cause confusion down the line, seeing a boolean string immediately looks to me like a bug. Would something like `value={include ? 'include' : 'exclude'}` be more clear? Or the state variable can could be a string instead of a boolean.

Reply:
Yeah I think that makes sense. This is sort of an annoyance of html radios and how they only accept string values. I will send a PR for this change